src/main/java/us/spaceclouds42/disableportals/commands/DisablePortalsCommand.java

```diff
@@ -123,10 +104,7 @@ private static int reloadConfig(ServerCommandSource source) {
         DisablePortals.CONF = Config.loadConfig(new File(FabricLoader.getInstance().getConfigDir() + "/DisablePortals.json"));
 
         source.sendFeedback(
-                new LiteralText("Reloaded the config file!").formatted(
-                        Formatting.GREEN
-                ),
-                false
+                Text.translatable("disableportals.reloadconfig").formatted(Formatting.GREEN), false
```

Comment:
Why did you convert these from literal texts to translatable texts? This is completely and entirely useless since translations are client-side

Reply:
Didn't realize there was no way (without this fabric language hack mod) to get translations server side..Kind of lame admins can't get a command in their native language sent to them.. guess I'll revert to the previous literals